### PR TITLE
Add sniff for collection aliases

### PR DESCRIPTION
**Pull request by magikid**
This adds a new sniff to check for usage of a collection method that is an alias of another method.

### CODE REVIEW

File: composer.json
Comment (magikid) [inline composer.json:1]: This was necessary to get the dependencies installed locally.

File: ruleset.xml [hunk 1]
```diff
@@ -106,8 +106,6 @@
             <property name="fixable" type="boolean" value="true"/>
         </properties>
     </rule>
-    <!-- Forbid dead code -->
-    <rule ref="SlevomatCodingStandard.Classes.UnusedPrivateElements"/>
```
Comment (magikid) [inline ruleset.xml]: No longer a thing.  [It's handled by PHPStan now](https://phpstan.org/blog/detecting-unused-private-properties-methods-constants).

File: ruleset.xml [hunk 2]
```diff
@@ -400,15 +398,15 @@
...
-            <property name="tokensToCheck" type="array">
-                <element value="T_IF" />
-                <element value="T_DO" />
-                <element value="T_WHILE" />
-                <element value="T_FOR" />
-                <element value="T_FOREACH" />
-                <element value="T_SWITCH" />
-                <element value="T_TRY" />
-                <element value="T_DEFAULT" />
+            <property name="controlStructures" type="array">
```
Comment (magikid) [inline ruleset.xml:401]: These all changed names in the update to 7.

**Comment by joanna-bak-sourceability** [comment]
looks good

**Pull request merged**